rubin_sim/maf/batches/scienceRadarBatch.py

```diff
@@ -596,6 +620,21 @@ def scienceRadarBatch(
         )
     )
 
+    metric = metrics.BDParallaxMetric(
```

Comment:
What is this reporting again? is it just the number of brown dwarfs that have parallax error less than a given amount?